### DISCUSSION TITLE
깃헙에 placeholder인 상태로 업로드 시 주석

### DISCUSSION THREAD

**Discussion by Yeggggi** [DeveloperAcademy-POSTECH/.github#21] [Idea]
placeholder 상태로 깃헙에 푸시하면 아래처럼 주석이 붙는다는걸 알고계셨나요?!
/*@START_MENU_TOKEN@*/ .blue /*@END_MENU_TOKEN@*/ 
신기해서 공유해요

<img width="731" alt="스크린샷 2024-05-29 오전 8 57 53" src="https://github.com/DeveloperAcademy-POSTECH/.github/assets/49176310/390d8f39-d1c9-43d0-a877-b13e2d84bd82">

**Comment by jinccc97** [comment]
'placeholder 상태로 깃헙에 푸시하면' 이 어떤건가요..?! 😵‍💫

**Comment by Yeggggi** [comment]
@jinccc97 

 placeholder는 문자나 값 등의 요소가 들어갈 자리에 임시로 채워져있는 내용물이예요!

예시를 들어보면 우리가 Text에 font size를 지정해주기 위한 설정을 해줄 때 .font라고 입력해서 Enter를 누르면 .title 이라는 임시값이 들어갑니다. 

우리는 이 임시값을 지우고 다른 값을 입력해주거나 Enter를 눌러서 .title 로 확정되어 넣어줍니다.
이때 Enter를 누르지 않고 임시로 값이 들어있는 상태를 placeholder라 하고 이 칩을 placeholder token 이라고 불러요!


![Frame 278](https://github.com/DeveloperAcademy-POSTECH/.github/assets/49176310/8155625e-2661-44af-90ac-0ba8cca7001b)

그래서 이렇게 값이 확정되기 전에 임시로 값이 들어있는 상태로 푸시를 하는 것을 'placeholder 상태로 깃헙에 푸시한다'고 표현했어요!

혹시 이것을 지칭하는 다른 말을 아시는 분이 계시다면 알려주시면 감사할 것 같아요 ·ᴗ·

**Comment by DevKang** [comment]
공유 감사합니다 ~!
시도해본 적 없는거라 신기하게 봤어요 👍 

참고로 저 상태로는 컴파일이 되지 않기 때문에 푸쉬된 코드를 누군가가 받았을 때 적지 않게 당황할 수도 있습니다 😅
